app/Console/Commands/SetUploadUsers.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\UploadUser;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Storage;
+
+class SetUploadUsers extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'uploadUsers:set {allowList}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Set the list of users who are allowed to upload mismatch files by providing a txt file;';
```

Comment:
This should probably say where the text file should be located/ how it should be provided.

Reply:
I added a description to the `allowlist` input argument. Try `sail artisan help uploadUsers:set` to see the help message.